mindsdb/interfaces/file/file_controller.py

```diff
@@ -104,7 +104,12 @@ def save_file(self, name, file_path, file_name=None):
             file_dir.mkdir(parents=True, exist_ok=True)
             source = file_dir.joinpath(file_name)
             # NOTE may be delay between db record exists and file is really in folder
-            shutil.move(file_path, str(source))
+            try:
+                shutil.move(file_path, str(source))
+            except PermissionError as e:
+                logger.warning('Permission error while moving file %s: %s', file_path, str(e))
```

Comment:
is it right not to raise an exception in this case?

Reply:
Could the cause of the problem be: the file description is still opened outside of this function?
Is [this function](https://github.com/mindsdb/mindsdb/blob/aea53cd15306a08a0e1a12a42051f2d18969cd5d/tests/unit/executor_test_base.py#L262) is previous in call stack?